dev.neovide.neovide.yml

```diff
@@ -0,0 +1,32 @@
+id: dev.neovide.neovide
+command: neovide
+runtime: org.freedesktop.Sdk
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+  - org.freedesktop.Sdk.Extension.rust-stable
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --talk-name=org.freedesktop.Flatpak
+  - --filesystem=host
+  - --share=network
+  - --env=TERM=xterm-256color
+modules:
+  - modules/host-spawn/host-spawn.yml
+  - modules/fd/fd.yml
+  - modules/ripgrep/ripgrep.yml
+  - modules/neovim/neovim.yml
+  - modules/neovide/neovide.yml
```

Comment:
i'd prefer if you have neovide in the main manifest as that's the main module.

Reply:
> i'd prefer if you have neovide in the main manifest as that's the main module.

i don't like it, but ok, done